spec/components/badges/view_preview.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+module Badges
+  class ViewPreview < ViewComponent::Preview
+    include FactoryBot::Syntax::Methods
+
+    def with_no_trainees_in_award_states
+      ActiveRecord::Base.transaction do
```

Comment:
I think this will create the data in your local db when you access the previews locally

Reply:
ah damn, yeah I meant to put a rollback in after the page was rendered 🤦